### PR TITLE
Include typings/tests.ts into typecheck

### LINKED ISSUE #767: 'npm run typecheck' ignores tests.ts
## Context

`npm run typecheck` (which is also run by the precommit hook) does not use `typings/test.ts` file. This could make it difficult to catch bugs or inconsistencies between the used code and typings.

* Telegraf.js Version: cad315b1573dcdc8d7bbcdb0bd2b2114e03b1519
* Node.js Version: v12.10.0
* Operating System: Ubuntu 16.04

## Expected Behavior

`typings/test.ts` is processed, all found errors are reported

## Current Behavior

`typings/test.ts` is not processed.

### Steps to Reproduce

Add

```typescript
const a: number = ['s'];
```

to `typings/test.ts`. Then run `npm run typecheck`. No errors are reported.

### PR DESCRIPTION
**Pull request by sjinks**
# Description

Run typecheck against `typings/tests.ts`

Fixes #767

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)

# How Has This Been Tested?

Tested by adding a bug to `typings/tests.ts` and then running `npm run typecheck`.

**Test Configuration**:
* Node.js Version: v12.10.0
* Operating System: Ubuntu 16.04

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation — _not applicable_
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: typings/test.ts [hunk 1]
```diff
@@ -1,14 +1,14 @@
 // This is a test file for the TypeScript typings.
 // It is not intended to be used by external users.
-import Telegraf, { Markup } from './index';
+import Telegraf, { Markup, Middleware, ContextMessageUpdate } from './index';
 
 
 const randomPhoto = 'https://picsum.photos/200/300/?random'
-const sayYoMiddleware = ({ reply }, next) => reply('yo').then(() => next())
+const sayYoMiddleware: Middleware<ContextMessageUpdate> = ({ reply }, next) => reply('yo').then(() => next && next())
```
Comment (sjinks) [inline typings/test.ts]: This fixes:
  * error TS7031: Binding element 'reply' implicitly has an 'any' type
  * error TS7006: Parameter 'next' implicitly has an 'any' type
  * error TS2722: Cannot invoke an object which is possibly 'undefined' (the definition of `Middleware` says that `next` is optional and can be `undefined`)

File: typings/test.ts [hunk 2]
```diff
@@ -1,14 +1,14 @@
...
 
 const {reply} =  Telegraf;
 
-const bot = new Telegraf(process.env.BOT_TOKEN)
+const bot = new Telegraf(process.env.BOT_TOKEN || '')
```
Comment (sjinks) [inline typings/test.ts]: Fixes error TS2769: No overload matches this call: `process.env.BOT_TOKEN` can be `undefined`, while the constructor expects a string.

File: typings/test.ts [hunk 3]
```diff
@@ -25,7 +25,7 @@ bot.command('foo', reply('http://coub.com/view/9cjmt'))
 
 bot.action('bar', reply('i was here'))
 
-bot.telegram.sendMessage(process.env.BOT_CLIENT_ID,"It's work")
+bot.telegram.sendMessage(process.env.BOT_CLIENT_ID || '', "It works")
```
Comment (sjinks) [inline typings/test.ts]: Fixes error TS2345: Argument of type 'string | undefined' is not assignable to parameter of type 'string | number'

**Comment by dotcypress** [comment]
@sjinks amazing job, thank you!

**Pull request merged**